docs/platforms/php/index.mdx

```diff
@@ -1,8 +1,9 @@
 ---
-title: PHP
+title: Configure the PHP SDK
 sdk: sentry.php
 caseStyle: snake_case
 supportLevel: production
+description: PHP (or Hypertext Preprocessor) is an open source general-purpose scripting language that's well-suited for web development. It can be embedded into HTML. 
```

Comment:
```suggestion
description: PHP (or Hypertext Preprocessor) is an open source, general-purpose scripting language that's well-suited for web development. It can be embedded into HTML. 
```